contracts/truefi/TruPriceChainlinkOracle.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.10;
+
+import {SafeMath} from "@openzeppelin/contracts/math/SafeMath.sol";
+import {ITruPriceOracle} from "./interface/ITruPriceOracle.sol";
+import {IUniswapPair} from "./interface/IUniswapPair.sol";
+
+interface IChainLink {
+    function latestAnswer() external view returns (int256);
+}
+
+contract TruPriceChainLinkOracle is ITruPriceOracle {
+    using SafeMath for uint256;
+
+    IChainLink public chainlinkOracle;
+
+    constructor(IChainLink _chainlinkOracle) public {
+        chainlinkOracle = _chainlinkOracle;
+    }
+
+    /**
+     * @dev converts from USD with 18 decimals to TRU with 8 decimals
```

Comment:
From the comment I would expect to see a factor of 10^10 somewhere - how does dividing/multiplying by only 100 accomplish this conversion?

Reply:
add comment to explain the math here

Reply:
resolved